Sources/StructuredQueries/Macros.swift

```diff
@@ -34,11 +34,13 @@ public macro Table(_ name: String? = nil) =
 ///   - representableType: A type that represents the property type in a query expression. For types
 ///     that don't have a single representation in SQL, like `Date` and `UUID`.
 ///   - primaryKey: The column is its table's auto-incrementing primary key.
+///   - databaseInitialized: The column has a default value and is not needed in an insert statement.
 @attached(accessor, names: named(willSet))
 public macro Column(
```

Comment:
One idea to simplify the diagnostics you added in the macro for nonsensical combinations of these two parameters would be to instead introduce an overload of `@Column` that simply takes the `databaseInitialized` parameter. That way it's impossible to invoke `@Column` with both parameters.